site/layouts/partials/sidebar-tree.html

```diff
@@ -100,7 +100,7 @@
           {{ end }}
 
           {{ if $showPage }}
-            <a class="td-sidebar-link td-sidebar-link__page {{ if $activePage }} active{{ end }}" id="{{ $mid }}" href="{{ .RelPermalink }}">{{ .LinkTitle }}</a>
+            <a class="td-sidebar-link td-sidebar-link__page {{ if $activePage }} active{{ end }}" id="{{ $mid }}" href="{{ .RelPermalink }}"> ➔ {{ .LinkTitle }}</a>
```

Comment:
how about we add a blank space before the " ➔" so it indents more ?

Reply:
or how about we leave the Sub Item ones Without any icons. just the Parent ones have a  ▀  and these sub items should have an Extra Blank Space so it indents more....could we try that first ?

Reply:
updated preview link.
https://deploy-preview-12040--kubernetes-sigs-minikube.netlify.app/docs/